charts/leonardo/templates/cronjob.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: "resource-validator-cron"
+  labels: {}
+spec:
+  schedule: "30 4 * * *"
+  startingDeadlineSeconds: 100
+  successfulJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: leonardo-ksa
+          containers:
+          - image: "{{ .Values.cronjob.imageRepository }}:{{ .Values.cronjob.imageTag }}"
+            name: "{{ .Values.cronjob.name }}"
+            command: ["/bin/bash", "-c"]
+            args:
+              - |
+                trap "touch /tmp/pod/main-terminated" EXIT
+                /opt/docker/bin/resource-validator --dryRun --all
+            env:
+            - name: JAVA_OPTS
+              value: -Dlogback.configurationFile=file:/opt/docker/resource-validator-config/logback.xml
+            - name: LEONARDO_DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: leonardo-db-creds
+                  key: username              
+            - name: LEONARDO_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: leonardo-db-creds
+                  key: password
+            - name: LEONARDO_PATH_TO_CREDENTIAL
+              value: /secrets/service-account.json
+            volumeMounts:
+            - name: leonardo-sa-json-file
+              mountPath: /secrets
+              readOnly: true
+            - mountPath: /tmp/pod
+              name: tmp-pod
+            - mountPath: /opt/docker/resource-validator-config
+              name: logback-config
+          - name: cloudsql-proxy
+            image: gcr.io/cloudsql-docker/gce-proxy:1.15
+            env:
+            - name: SQL_INSTANCE_PROJECT
+              valueFrom:
+                secretKeyRef:
+                  name: leonardo-db-creds
+                  key: instance_project
+            - name: SQL_INSTANCE_REGION
+              valueFrom:
+                secretKeyRef:
+                  name: leonardo-db-creds
+                  key: instance_region
+            - name: SQL_INSTANCE_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: leonardo-db-creds
+                  key: instance_name
+            command: ["/bin/sh", "-c"]
+            args:
+            - |
```

Comment:
Add comment indicating why this workaround is here & that it should be removed once sidecar lifecycle support is available?